site/src/content/docs/forms/checkbox.mdx

```diff
@@ -360,19 +421,19 @@ When checkboxes belong to a group (e.g., in a form), you must provide clear cont
 
 <Callout type="warning" name="standalone-form-indicator" />
 
-For the standalone Checkbox, we provide a completely different architecture to ease the integration inside your projects.
+For the standalone checkbox, we provide a completely different architecture to ease the integration inside your projects.
```

Comment:
I think we talk about the component here so I'd let the Maj in here but open to discuss

Reply:
in that case, shouldn't we use "OUDS Checkbox" ? But do we have to ?